lighty-modules/lighty-gnmi/lighty-gnmi-connector/src/test/java/io/lighty/modules/gnmi/connector/tests/GnmiTest.java

```diff
@@ -6,7 +6,7 @@
  * and is available at https://www.eclipse.org/legal/epl-v10.html
  */
 
-package io.lighty.modules.southbound.gnmi.tests;
+package io.lighty.modules.gnmi.connector.tests;
```

Comment:
package io.lighty.gnmi.connector.gnmi is better as you also changed the GnoiTest to package io.lighty.gnmi.connector.gnoi